### ISSUE TITLE
[BUG] rancher-monitoring-grafana is too small

### ISSUE THREAD

**Issue by abonillabeeche** [harvester/harvester#3307]
The size of 10M is constantly creating issues because the content may exceed the disk and the pod won't start unless the PV is cleared.

> p3610:/var/lib/harvester/defaultdisk/replicas/pvc-c9c9e551-d216-411c-a508-a50b5e0664b5-a83bfb0b # ls -l
> total 10256
> -rw------- 1 root root       4096 Oct 31 04:39 revision.counter
> -rw-r--r-- 1 root root 5368709120 Oct 31 04:49 volume-head-001.img
> -rw-r--r-- 1 root root        159 Oct 31 04:49 volume-head-001.img.meta
> -rw-r--r-- 1 root root   10485760 Oct 31 04:39 volume-snap-expand-5368709120.img
> -rw-r--r-- 1 root root        156 Oct 31 04:49 volume-snap-expand-5368709120.img.meta
> -rw-r--r-- 1 root root        175 Dec 21 17:01 volume.meta
> p3610:/var/lib/harvester/defaultdisk/replicas/pvc-c9c9e551-d216-411c-a508-a50b5e0664b5-a83bfb0b # du -h
> 11M	.
> p3610:/var/lib/harvester/defaultdisk/replicas/pvc-c9c9e551-d216-411c-a508-a50b5e0664b5-a83bfb0b # rm *
> p3610:/var/lib/harvester/defaultdisk/replicas/pvc-c9c9e551-d216-411c-a508-a50b5e0664b5-a83bfb0b # ls

Please set to priority0 for 1.1.2

**Comment by bk201** [comment]
Related: https://github.com/harvester/harvester/issues/2724 
Expanding the volume should fix this. Please see if the doc helps: https://docs.harvesterhci.io/v1.1/troubleshooting/monitoring#expand-pvvolume-size

**Comment by w13915984028** [comment]
The default PVC size of grafana is expanded to 2GiB in v1.1.0  (https://github.com/harvester/harvester/issues/2282). 

For existing clusters, the size is not automatically changed when upgrading. User needs to change the size manually as mentioned above.

**Comment by connorkuehl** [comment]
The above commit (https://github.com/harvester/harvester-installer/pull/327/commits/c71c92788ddf50f91552af55a090636e5ef82da7) has been released since v1.1.0.

```
~/sources/harvester-installer$ git tag --contains a5a3846cbff110f5eb87dc95853ee3981860ae49 | sort --version-sort
v1.1.0
v1.1.0-rc1
v1.1.0-rc2
v1.1.0-rc3
v1.1.0-rc4
v1.1.1
v1.1.1-rc1
v1.1.1-rc2
v1.1.2
v1.1.2-rc1
v1.1.2-rc2
v1.1.2-rc3
v1.1.2-rc4
v1.1.2-rc5
v1.1.2-rc6
v1.1.2-rc7
v1.1.2-rc8
v1.1.2-rc9
v1.2.0-rc1
v1.2.0-rc2
v1.2.0-rc3
v1.2.0-rc4
```

I'll go ahead and close this, but if this is till impacting anyone, please feel free to re-open! 🙂
